Add a new event - backend.auth.afterLogin

filipac opened the pull request:
I wanted to listen to an event when a users logs-in and I realised that October does not yet have that. It would be great to support this out of the box to avoid users make changes to core files or listen to hack-ish events to do something after login.

tschallacka commented:
+1

daftspunk commented:
I think you want this to go in the Backend User model? See RainLab.User for example.

filipac commented:
I opened a new PR on backend module, i moved the event (and renamed it) to apply only on the backend User model. https://github.com/octobercms/october/compare/master...filipac:patch-1
